index.js

```diff
@@ -713,6 +713,12 @@
   //. Type comprising every primitive String value.
   var String_ = NullaryTypeWithUrl('String', typeofEq('string'));
 
+
+  //# Symbol :: Type
+  //.
+  //. Type comprising every primitive Symbol value.
+  var Symbol_ = NullaryTypeWithUrl('Symbol', typeofEq('symbol'));
```

Comment:
Is there a reason to favour `typeofEq('symbol')` over `typeEq('Symbol')`, or are the expressions equivalent?